src/index.html

```diff
@@ -9,6 +9,11 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="clock">
+      <div class="clock__hours"></div>
```

Comment:
The class name 'clock__hours' is not following BEM naming convention. It should be something like 'clock__hand--hour' since it's a modifier of the 'hand' element in the 'clock' block.